cql3/query_processor.cc

```diff
@@ -763,7 +763,7 @@ std::pair<std::reference_wrapper<struct query_processor::remote>, gate::holder>
 
 query_options query_processor::make_internal_options(
```

Comment:
nit: commit message:
> data_value_list is a wrapper around std::initializer_list<data_value>.

doesn't correspond to the code (`data_value_list` is defined to be equal to `std::initializer_list<data_value>;` in this commit)